.renovaterc.json

```diff
@@ -10,6 +10,11 @@
     "group:linters",
     "schedule:nonOfficeHours"
   ],
+  "allowedPostUpgradeCommands": [
```

Comment:
is this working for github ? 
I thought this was only for the config of the self hosted renovate bot

also, don't you need to add `yarn ng update` and `yarn dlx` as well ?

Reply:
I did not check the doc and has been misled by the linked PR messages.
Actually you are right, this options is only for self-hosted Renovate (cf: https://docs.renovatebot.com/self-hosted-configuration/#allowpostupgradecommandtemplating).
Then I do not know how to fix the current issue :S